docs/tox.ini

```diff
@@ -0,0 +1,20 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+[tox]
+work_dir = {toxinidir}/../.tox
```

Comment:
Is there any reason to have the `.tox` directory at the top level, rather than be in `docs/.tox`?

Reply:
If `.tox` itself is in `docs`, sphinx will find the READMEs that come from those packages installed in the tox environment and throw a bunch of warnings.

Reply:
I moved it back to `docs/` and used `exclude_patterns` to avoid building the `.tox` folder.